docker-compose.s3.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+services:
+    #
+    # mender-deployments using s3
+    #
+    mender-deployments:
+        image: mendersoftware/deployments:latest
+        extends:
+            file: common.yml
+            service: mender-base
+        networks:
+            - mender
+        environment:
+            AWS_ACCESS_KEY_ID: %AWS_KEY%
```

Comment:
${AWS_KEY} ${AWS_SECRET} will work too right? Those %..% remind me of windows, bleh.